pkgs/kde/gear/kdenlive/default.nix

```diff
@@ -14,6 +15,8 @@
   libv4l,
   open-timeline-io,
   frei0r,
+  rnnoise-plugin,
+  enablePlugins ? true,
```

Comment:
Would it make sense to add a generic way to add plugins to kdendlive?

Reply:
In the absence of any actual user who wants to customize this more, and who isn't happy with one of the following
* using an override to set `enablePlugins = false`
* seperate `kdenlive`/`kdenliveWithPlugins`
* seperate `kdenliveMinimal` / `kdenlive` packages


I fully agree with this take: https://github.com/NixOS/nixpkgs/pull/402815#issuecomment-2853935530

I would not have the time to write let alone maintain a way to add plugins like that, especially since different plugins need to be added to different environment variables.

IMO there is such a thing a s too much configurabiliy, if nobody needs it, and it just creates a maintenance burden for no reason.